handler/process/adaptor.go

```diff
@@ -96,6 +96,7 @@ func (a *Adaptor) HandleMessage(
 		packer:     a.Packer,
 		logger:     a.Logger,
 		instanceID: inst.InstanceID,
+		ended:      inst.HasEnded,
```

Comment:
This may not be necessary, as you pointed out, adding it doesn't change any behavior. If we _do_ need more test cases, it would be to guarantee that both events and timeouts are ignored for ended processes (these may already exist?)

Reply:
See https://github.com/dogmatiq/verity/pull/665#issuecomment-3099322057